packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-policy-drilldown.vue

```diff
@@ -426,6 +453,12 @@ const onSaveInterventionPolicy = async () => {
 	}
 };
 
+const createNewInterventionPolicy = () => {
+	if (!model.value?.id) return;
+	newBlankInterventionPolicy.value.modelId = model.value?.id;
```

Comment:
You just tested that it was not null or undefined
```suggestion
	newBlankInterventionPolicy.value.modelId = model.value.id;
```